src/patternfly/components/NavSystem/styles.scss

```diff
@@ -0,0 +1,481 @@
+  // color
+  --pf-c-nav__list-link--Color:                               var(--pf-global--Color--dark-100);
+  --pf-c-nav__list-link--active--Color:                       var(--pf-global--link--Color);
+  --pf-c-nav__list-link--disabled--Color:                     var(--pf-global--disabled-color--100);
+  // background
+  --pf-c-nav__list-link--disabled--BackgroundColor:           var(--pf-global--disabled-color--200);
+  // after background
+  --pf-c-nav__list-link--after--BackgroundColor:              transparent;
+  --pf-c-nav__list-link--active--after--BackgroundColor:      var(--pf-global--link--Color);
+  --pf-c-nav__list-link--disabled--after--BackgroundColor:    transparent;
+  // hardcoding border bottom to match design spec
+  --pf-c-nav__list-link--after--Width:                        3.125rem;
+  --pf-c-nav__list-link--after--Height:                       3px;
+
+  // List list action
+  --pf-c-nav__list-expand--PaddingRight:                      var(--pf-global--spacer--sm);
+  --pf-c-nav__list-expand--PaddingLeft:                       var(--pf-global--spacer--sm);
+
+
+  // List simple link
+  --pf-c-nav__list-simple-link--PaddingTop:                   var(--pf-global--spacer--sm);
+  --pf-c-nav__list-simple-link--PaddingRight:                 var(--pf-global--spacer--xl);
+  --pf-c-nav__list-simple-link--PaddingBottom:                var(--pf-global--spacer--sm);
+  --pf-c-nav__list-simple-link--PaddingLeft:                  var(--pf-global--spacer--xl);
+  // font
+  --pf-c-nav__list-simple-link--FontWeight:                   var(--pf-global--FontWeight--normal);
+  --pf-c-nav__list-simple-link--active--FontWeight:           var(--pf-global--FontWeight--semi-bold);
+  // color
+  --pf-c-nav__list-simple-link--Color:                        var(--pf-global--Color--dark-100);
+  --pf-c-nav__list-simple-link--active--Color:                var(--pf-global--link--Color);
+  --pf-c-nav__list-simple-link--disabled--Color:              var(--pf-global--disabled-color--100);
+  // background
+  --pf-c-nav__list-simple-link--active--BackgroundColor:      $pf-color-black-150;
+  --pf-c-nav__list-simple-link--disabled--BackgroundColor:    var(--pf-global--disabled-color--200);
+
+
+  // List horizontal link
+  --pf-c-nav__list-horizontal-item--MarginRight:                        var(--pf-global--spacer--xl);
+  --pf-c-nav__list-horizontal-link--PaddingTop:                         var(--pf-global--spacer--lg);
+  --pf-c-nav__list-horizontal-link--PaddingBottom:                      var(--pf-global--spacer--lg);
+  --pf-c-nav__list-horizontal-link--FontWeight:                         var(--pf-global--link--normal);
+  --pf-c-nav__list-horizontal-link--active--FontWeight:                 var(--pf-global--FontWeight--bold);
+  // color
+  --pf-c-nav__list-horizontal-link--Color:                              $pf-color-black-300;
+  --pf-c-nav__list-horizontal-link--active--Color:                      $pf-color-blue-300;
```

Comment:
should this be `var(--pf-global--active-color--300)`?

Reply:
Updated